Update/wata_tips_11.txt

```diff
@@ -82,10 +82,10 @@ void main()
 	if (GetGlobalFlag(GADVMode)) {
 		OutputLineAll("", NULL, Line_ContinueAfterTyping);
 		OutputLine(NULL, "　以下は仮説だが、綿流しの儀は、閉鎖環境である鬼ヶ淵村の内側に何らかの不満が募った場合のはけ口や、目を逸らすための目的で開かれた、",
-			   NULL, "<size=-2>This is just a hypothesis, but maybe the rite of Watanagashi was a political event for the purpose either being an outlet for (or outright diverting the attention of) the villagers of the isolated Onigafuchi Village, ", GetGlobalFlag(GLinemodeSp));
+			   NULL, "<size=-2>This is just a hypothesis, but maybe the rite of Watanagashi was a political event for the purpose of either being an outlet for, or outright diverting the attention of the villagers of the isolated Onigafuchi Village, ", GetGlobalFlag(GLinemodeSp));
```

Comment:
I think we should keep the brackets here. They make the meaning more clear imo.

Reply:
I'd agree, but the brackets are meant for something added in the sentence, that can be overlooked and still have a full sentence. Which is not the case here, since it's written "either" but there's only one path without the parenthesis. It was a bit of a puzzleknot

Reply:
Sounds good.